garnett/expressions.py

```diff
@@ -1,24 +1,23 @@
 from django.db.models import F
 from django.db.models.fields.json import KeyTextTransform
 
-from garnett.utils import get_current_language
+from garnett.utils import get_current_language_code
 from garnett.fields import TranslatedField
 
 
 # Based on: https://code.djangoproject.com/ticket/29769#comment:5
 # Updated comment here
 # https://code.djangoproject.com/ticket/31639
 class LangF(F):
-    def resolve_expression(
-        self, query=None, allow_joins=True, reuse=None, summarize=False, for_save=False
-    ):
-        rhs = super().resolve_expression(query, allow_joins, reuse, summarize, for_save)
+    def resolve_expression(self, *args, **kwargs):
+        print(args, kwargs)
```

Comment:
should we use `logging.debug` instead of `print`?

or other flag to make this `print` not work at production.

e.g. `settings.DEBUG` in Django

Reply:
Hi @shabbywu, thanks for your review. This PR is an early state, and we'll probably remove the `print` statement entirely. Hope that helps.